test/powershell/engine/Job/Jobs.Tests.ps1

```diff
@@ -69,6 +69,14 @@ Describe 'Basic Job Tests' -Tags 'CI' {
             $ProgressMsg[0].StatusDescription | Should -BeExactly 2
         }
 
+        It 'Can use the user specified working directory parameter'{
+            $parent = [System.IO.Path]::GetTempPath()
+            $job = Start-Job -ScriptBlock { $pwd } -WorkingDirectory $parent| Wait-Job
+            Wait-Job $job
+            Receive-Job $job -OutVariable jobOutput
```

Comment:
Is there a reason you wrote it this way? I would usually opt for something more straightforward, like...
```suggestion
            $jobOutput = Receive-Job $job
```

Reply:
@vexx32 thanks for the review
fixed in https://github.com/davinci26/PowerShell/commit/06ef81a76db4898ec1066f69e50558d47f8e85f0